package.json

```diff
@@ -130,5 +130,5 @@
       ".*": "babel-jest"
     }
   },
-  "sideEffects": true
+  "sideEffects": false
```

Comment:
do you need to include it if false? What's the default value?

Reply:
it was set to false previously, I set it to true when I tested storybook build, and forgot to change that 🙈

Reply:
because of that we had errors in bundle size checks when bumping style guide version in brainly-frontend (it was importing more things than needed)

Reply:
@anna-wro default value is `true`. It can have false but also an array which files have side effects and which don't.